example/spec/parallel/syncGranule/SyncGranuleSuccessSpec.js

```diff
@@ -156,53 +159,4 @@ describe('When the Sync Granules workflow is configured to overwrite data with d
       expect(record.status).toEqual('completed');
     });
   });
-
-  describe('and it encounters data with a duplicated filename', () => {
-    let lambdaOutput;
-    let existingfiles;
-    let fileUpdated;
-
-    beforeAll(async () => {
-      lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'SyncGranule');
-      const files = lambdaOutput.payload.granules[0].files;
-      existingfiles = await getFilesMetadata(files);
-
-      // update one of the input files, so that the file has different checksum
-      const content = randomString();
-      const file = inputPayload.granules[0].files[0];
-      fileUpdated = file.name;
-      const updateParams = {
-        Bucket: config.bucket, Key: path.join(file.path, file.name), Body: content
-      };
-
-      await s3().putObject(updateParams).promise();
-      inputPayload.granules[0].files[0].fileSize = content.length;
-
-      workflowExecution = await buildAndExecuteWorkflow(
-        config.stackName, config.bucket, workflowName, collection, provider, inputPayload
-      );
-    });
-
-    it('does not raise a workflow error', () => {
-      expect(workflowExecution.status).toEqual('SUCCEEDED');
-    });
-
-    it('overwrites the existing file with the new data', async () => {
```

Comment:
Did this test get moved or do we just not need it?

Reply:
Moved. I put all the dup granule handling tests in one spec.

Reply:
It doesn't seem like there is an explicit integration test for this exact case anymore, but there are unit tests for both `sync-granule` and `move-granules` of this case, so that's probably fine